docs/index.html

```diff
@@ -0,0 +1,58 @@
+<h1>Lockbox for desktop</h1>
```

Comment:
Why are we using HTML here? I don't see any particular reason it would be necessary...

Reply:
@jimporter I found the inflexibility in applying styles to markdown hard to achieve the side-by-side layout we have here. How would you structure the left and right side of the page with markdown and CSS to satisfy the "right callout" layout?

Reply:
I'd just use [admonitions](http://pythonhosted.org/Markdown/extensions/admonition.html) and add some styles for a "right" admonition.

Reply:
>I'd just use admonitions and add some styles for a "right" admonition.

TIL! I'll make that change right now. Thanks! 🙏

Reply:
One caveat: you probably won't get the "main" content to remain narrow below the floating admonition, but with appropriate margins, I think that'll be ok.